src/cluster.c

```diff
@@ -101,6 +103,10 @@ static void clusterBuildMessageHdr(clusterMsg *hdr, int type, size_t msglen);
 #define dictEntryPrevInSlot(de) \
     (((clusterDictEntryMetadata *)dictEntryMetadata(de))->prev)
 
+#define isSlotOwnershipSpeculative(slot) \
+    (server.cluster->slots[slot] == NULL || \
+        bitmapTestBit(server.cluster->owner_not_owning_slot, slot))
+
```

Comment:
```suggestion
#define isSlotUnclaimed(slot) \
    (server.cluster->slots[slot] == NULL || \
        bitmapTestBit(server.cluster->owner_not_owning_slot, slot))
```
Minor preference since I think this makes the intent very clear, I do like the macro, it helps a lot.

Reply:
I like this name better 👍